llvm/include/llvm/Analysis/LoopAccessAnalysis.h

```diff
@@ -216,6 +216,12 @@ class MemoryDepChecker {
     return MaxSafeVectorWidthInBits;
   }
 
+  /// Return safe power-of-2 number of elements, which do not prevent store-load
+  /// forwarding and safe to operate simultaneously.
```

Comment:
```suggestion
  /// Return safe power-of-2 number of elements, which do not prevent store-load
  /// forwarding.
```

Reply:
Also comment says this returns the number of elements, but it returns the number of bits size which does not prevent store-load forwarding?